tests/unit/combine_results/expected_output.txt

```diff
@@ -1 +1,2 @@
 ./tests/unit/combine_results/combined_MotEvo_results.tsv
+./tests/unit/ProbabilityvsSequences.pdf
```

Comment:
Same as above - Why is this file under a unit test for the `combine_results` rule? This file is not an output of `combine_results` but a step after that. Therefore it need to have its own directory with its separate input, output and md5-related files.